ACPs/77-reinventing-subnets/README.md

```diff
@@ -510,66 +511,67 @@ def cost_over_time(V:int, T:int, x:int, Δt: int) -> int:
 
 Any state execution changes must be coordinated through a mandatory upgrade. Implementors must take care to continue to verify the existing ruleset until the upgrade is activated. After activation, nodes should verify the new ruleset. Implementors must take care to only verify the presence of 2000 $AVAX prior to activation.
 
 ### Deactivated Transactions
 
 - P-Chain
+
   - `TransformSubnetTx`
 
     After this ACP is activated, Elastic Subnets will be disabled. `TransformSubnetTx` will not be accepted post-activation. As there are no Mainnet Elastic Subnets, there should be no production impact with this deactivation.
 
 ### New Transactions
 
 - P-Chain
-  - `ConvertSubnetTx`
-  - `RegisterSubnetValidatorTx`
-  - `SetSubnetValidatorWeightTx`
-  - `DisableValidatorTx`
-  - `IncreaseBalanceTx`
+  - `ConvertL1Tx`
+  - `RegisterL1ValidatorTx`
+  - `SetL1ValidatorWeightTx`
+  - `DisableL1ValidatorTx`
+  - `IncreaseL1ValidatorBalanceTx`
 
 ## Reference Implementation
 
 A full reference implementation has not been provided yet. It will be provided once this ACP is considered `Implementable`.
 
 ## Security Considerations
 
-This ACP significantly reduces the cost of becoming a Subnet Validator. This can lead to a large increase in the number of Subnet Validators going forward. Each additional Subnet Validator adds consistent RAM usage to the P-Chain. However, this should be appropriately metered by the continuous fee mechanism outlined above.
+This ACP significantly reduces the cost of becoming an L1 validator. This can lead to a large increase in the number of L1 validators going forward. Each additional L1 validator adds consistent RAM usage to the P-Chain. However, this should be appropriately metered by the continuous fee mechanism outlined above.
 
-With the additional sovereignty Subnets gain from the P-Chain, Subnet staking tokens are no longer locked on the P-Chain for Permissionless Subnets. This poses a new security consideration for Subnet Validators: Malicious Subnets can choose to remove validators at will and take any funds that the Subnet Validator has on the Subnet. The P-Chain only provides the guarantee that Subnet Validators can retrieve the remaining $AVAX Balance for their Validator via a `DisableValidatorTx`. Any assets on the Subnet is entirely under the purview of the Subnet. The onus is now on Subnet Validators to vet the Subnet's security.
+With the additional sovereignty L1s gain from the P-Chain, L1 staking tokens are no longer locked on the P-Chain for sovereign L1s. This poses a new security consideration for L1 validators: Malicious L1s can choose to remove validators at will and take any funds that the L1 validator has on the L1. The P-Chain only provides the guarantee that L1 validators can retrieve the remaining $AVAX Balance for their Validator via a `DisableL1ValidatorTx`. Any assets on the L1 are entirely under the purview of the L1. The onus is now on L1 validators to vet the L1's security.
```

Comment:
As an example of readability, "With the additional sovereignty L1s gain" doesn't really work since L1s didn't exist prior to this PR. The lift to rework this ACP to accomodate this nomenclature change will be fairly high.